dask_image/Dockerfile

```diff
@@ -30,5 +31,4 @@ RUN chmod -R ugo+w /opt/conda \
     && conda clean -tipy \
     && chmod -R ugo+w /opt/conda
```

Comment:
once `dask` is no longer using Jenkins, all of these `chmod` commands can be removed across the Dockerfiles.

`chmod` artificially inflates the size of Docker images: https://medium.com/@lmakarov/the-backlash-of-chmod-chown-mv-in-your-dockerfile-f12fe08c0b55